### DISCUSSION TITLE
Per-site ad-blocking manual discrimination

### DISCUSSION THREAD

**Discussion by Ravenant1234** [uBlockOrigin/uBlock-issues#2728] [Ideas]
How about implementing a per-site ad-blocking manual discrimination? 
I will be more specific, what if i know wich type of ads a website load and i didnt want to load all filter lists for that site (only specific ones that i am defining myself?)
it may be useful to add a feature that let UBlock export a list of all content blocked in a website to add it in a site-specific list to only load that list (**and no other than that list**) anytime that site is accessed boosting-up the speed of browsing

**Comment by gothic-bum** [comment]
Disabling filter lists on a per-site basis was previously suggested and declined:
https://github.com/uBlockOrigin/uBlock-issues/issues/267
